docs/src/user-guide/formatters/index.md

```diff
@@ -211,6 +253,10 @@ not ok 1 - /var/lib/jenkins/workspace/Releases/eslint Release/eslint/fullOfProbl
 
 ### unix
 
+TODO: quick overview
```

Comment:
This is another common format.

Context: https://github.com/eslint/eslint/issues/2991